Sources/NIO/ChannelOption.swift

```diff
@@ -72,17 +72,37 @@ extension ChannelOptions {
         public struct SocketOption: ChannelOption, Equatable {
             public typealias Value = (SocketOptionValue)
 
-            public var level: SocketOptionLevel
-            public var name: SocketOptionName
+            public var optionLevel: NIOBSDSocket.OptionLevel
+            public var optionName: NIOBSDSocket.Option
+
+            public var level: SocketOptionLevel {
+                return SocketOptionLevel(optionLevel.rawValue)
+            }
+            public var name: SocketOptionName {
+                return SocketOptionName(optionName.rawValue)
+            }
```

Comment:
These two need setters as well to be API compliant.

Reply:
Right, added.  I'm just dumb.